modules/user_accounts/test/user_accountsTest.php

```diff
@@ -204,6 +204,9 @@ function testAddNewUser()
         $field->clear();
         $field->sendKeys('email@example.com');
         $this->safeClick(WebDriverBy::Name('SendEmail'));
+        $field = $this->safeFindElement(WebDriverBy::Name('CenterIDs[]'));
```

Comment:
207-209
        $SitesElement =  $this->safeFindElement(
                                  WebDriverBy::Name('CenterIDs[]')
                               );
        $SitesOption = new WebDriverSelect($SitesElement);
        $SitesOption->selectByValue("1");